Polishchuk_Dariia_HW2(1 sum and avarege ).cpp

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+
+int main() {
+  size_t a_size;
+  long sum;
+  std::cout << "Enter the number of numbers: ";
+  std::cin >> a_size;
+  int a[a_size];
+  for (int i = 0; i < a_size; ++i) {
```

Comment:
```suggestion
  for (size_t i = 0; i < a_size; ++i) {
```
тут лучше указать что счетчик и условие должны быть одного типа чтобы не было например переполнения счетчика и вы не получили вечный цикл